model-optimizer/extensions/ops/ExtractImagePatches.py

```diff
@@ -43,7 +43,8 @@ def infer(node: Node):
 
         assert len(input_shape) == 4, 'ExtractImagePatches operation supports only 4D tensors'
 
-        layout = node.graph.graph['layout']
+        # layout = node.graph.graph['layout']
+        layout = 'NHWC'
```

Comment:
Is it correct that layout is always set to NCHW?

Reply:
Yes, it is correct, because, according to @lazarevevgeny, the operation `ExtractImagePatches` should **always** calculate output shape in NHWC layout (because in OV the operation produces the tensor in NHWC layout).

Reply:
Okay, so can you remove the commented code line above and add comment with description why we set layout to NHWC.

Reply:
Done.